x-pack/legacy/plugins/uptime/public/components/functional/monitor_list/monitor_list.tsx

```diff
@@ -111,7 +111,10 @@ export const MonitorListComponent = (props: Props) => {
             values: { length: items.length },
           })}
           error={errors ? formatUptimeGraphQLErrorList(errors) : errors}
-          loading={loading}
+          // Only set loading to true when there are no items present to prevent the bug outlined in
+          // in https://github.com/elastic/eui/issues/2393 . Once that is fixed we can simply set the value here to
+          // loading={loading}
+          loading={loading && (!items || items.length < 1)}
           isExpandable={true}
```

Comment:
Great Catch, i thought it's much more complex, but i was looking from an eye of React Developer :)
Sometimes it's obvious yet hidden.